src/simularium/VisGeometry.ts

```diff
@@ -370,14 +371,68 @@ class VisGeometry {
         } else {
             this.resetBounds(DEFAULT_VOLUME_DIMENSIONS);
         }
-        if (this.resetCameraOnNewScene) {
-            this.resetCamera();
+
+        // Get default camera transform values from data
+        if (trajectoryFileInfo.cameraDefault) {
+            this.cameraDefault = trajectoryFileInfo.cameraDefault;
+        } else {
+            this.logger.warn(
+                "Using default camera settings since none were provided"
+            );
+            this.cameraDefault = DEFAULT_CAMERA_SPEC;
```

Comment:
small caution here: if we modify this.cameraDefault, then we run the risk of modifying DEFAULT_CAMERA_SPEC.  Could either clone or use some version of Object.freeze combined with readOnly to enforce it better?

Reply:
😱 Today I Learned JS objects are reference types..... I used lodash.deepClone to fix it. Thanks for catching it. Now I'm wondering how many lines of code I've written that's got this flaw in it has ever been merged 😬